frame/babel/Cargo.toml

```
@@ -13,6 +13,7 @@ workspace = true
 
 [dependencies]
 bech32 = { workspace = true, optional = true }
+bincode = { workspace = true }
```

```suggestion
bincode = { workspace = true, optional = true }
```
- Add `bincode` to the feature `solana`.
- Add `bincode?/std` to the feature "std".